pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -300,22 +313,33 @@ func (cl *candidateList) get() []Candidate {
 	return cl.items[:cl.size()]
 }
 
+// nodeToStatusMap is used to record node status.
+type nodeToStatus struct {
```

Comment:
we don't need this type, the mutex can be defined as a local variable inside dryRunPreemption.

Reply:
Done